test/common/quic/envoy_quic_session_cache_test.cc

```diff
@@ -31,9 +31,10 @@ std::vector<uint8_t> createFakeStatelessResetToken() {
 std::unique_ptr<quic::TransportParameters> makeFakeTransportParams() {
   auto params = std::make_unique<quic::TransportParameters>();
   params->perspective = quic::Perspective::IS_CLIENT;
-  params->version = FakeVersionLabel;
-  params->supported_versions.push_back(FakeVersionLabel);
-  params->supported_versions.push_back(FakeVersionLabel2);
+  params->legacy_version_information = quic::TransportParameters::LegacyVersionInformation();
```

Comment:
Oh, interesting. I'm slightly surprised that this is required because I thought it was optional. But I guess it's going to be optional eventually, just not yet?

Reply:
It is optional indeed. But this test uses this field.

Reply:
`legacy_version_information` is `absl::optional` but our code (currently) requires it to be present - we'll remove it at a later date. @danzh2010's change is the correct one here.